pkg/codegen/operations.go

```diff
@@ -510,10 +510,12 @@ func OperationDefinitions(swagger *openapi3.T) ([]OperationDefinition, error) {
 					return nil, fmt.Errorf("error generating default OperationID for %s/%s: %s",
 						opName, requestPath, err)
 				}
-				op.OperationID = op.OperationID
 			} else {
 				op.OperationID = ToCamelCase(op.OperationID)
```

Comment:
Please see the `typeNamePrefix` function in utils.go - it can do something like this automatically, combined with `SanitizeGoIdentity`.